source/blog/2017-03-14-a-tale-of-tempest-rpm-with-installers.html.md

```diff
@@ -0,0 +1,71 @@
+---
+title: A tale of Tempest rpm with Installers
+author: chandankumar
+tags: openstack,rdo,tempest,python-tempestconf
+date: 2017-03-14 18:27:02 IST
+---
+
+[Tempest](https://docs.openstack.org/developer/tempest/overview.html) is a set of integration tests to run against OpenStack Cloud.
+Delivering robust and working OpenStack cloud is always challenging. To make sure what we deliver in RDO is rock-solid, we use Tempest
+to perform a set of API and scenario tests against a running cloud using different installers like [puppet-openstack-integration](http://git.openstack.org/cgit/openstack/puppet-openstack-integration/),
+[packstack](http://git.openstack.org/cgit/openstack/packstack), and [tripleo-quickstart](http://git.openstack.org/cgit/openstack/tripleo-quickstart).
+And, it is the story of how we integrated RDO Tempest RPM package with installers so it can be consumed by various CI rather than using raw upstream sources.
+
+And the story begins from here:
+
+In RDO, we deliver Tempest as an rpm to be consumed by anyone to test their cloud. Till Newton release,
+we maintained a fork of [Tempest](https://github.com/redhat-openstack/tempest) which contains the config_tempest.py script to auto generate tempest.conf
```

Comment:
I know you wrote about the config_tempest.py before, you can make a link for the post